js/scripts.js

```diff
@@ -7,137 +7,160 @@ let bodyElement = document.querySelector("#body");
+				addTeamLinksToUi(item, buildTeamLinks(item));
+				return item;
 			});
 		})
 		.catch((err) => console.error(err));
-};
-
-teamButton.addEventListener("click", teamButtonAction);
-
-let boston = document.querySelector("#nba134860");
-boston.addEventListener("click", locateTeamData);
-
-let brooklyn = document.querySelector("#nba134861");
-brooklyn.addEventListener("click", locateTeamData);
-
-let newYork = document.querySelector("#nba134862");
-newYork.addEventListener("click", locateTeamData);
-
-let philadelphia = document.querySelector("#nba134863");
-philadelphia.addEventListener("click", locateTeamData);
-
-let toronto = document.querySelector("#nba134864");
-toronto.addEventListener("click", locateTeamData);
-
-let chicago = document.querySelector("#nba134870");
-chicago.addEventListener("click", locateTeamData);
-
-let cleveland = document.querySelector("#nba134871");
-cleveland.addEventListener("click", locateTeamData);
-
-let detroit = document.querySelector("#nba134872");
-detroit.addEventListener("click", locateTeamData);
-
-let indiana = document.querySelector("#nba134873");
-indiana.addEventListener("click", locateTeamData);
-
-let milwaukee = document.querySelector("#nba134874");
-milwaukee.addEventListener("click", locateTeamData);
-
-let atlanta = document.querySelector("#nba134880");
-atlanta.addEventListener("click", locateTeamData);
-
-let charlotte = document.querySelector("#nba134881");
-charlotte.addEventListener("click", locateTeamData);
-
-let miami = document.querySelector("#nba134882");
-miami.addEventListener("click", locateTeamData);
-
-let orlando = document.querySelector("#nba134883");
-orlando.addEventListener("click", locateTeamData);
-
-let washington = document.querySelector("#nba134884");
-washington.addEventListener("click", locateTeamData);
-
-let denver = document.querySelector("#nba134885");
-denver.addEventListener("click", locateTeamData);
-
-let minnesota = document.querySelector("#nba134886");
-minnesota.addEventListener("click", locateTeamData);
-
-let oklahoma = document.querySelector("#nba134887");
-oklahoma.addEventListener("click", locateTeamData);
-
-let portland = document.querySelector("#nba134888");
-portland.addEventListener("click", locateTeamData);
-
-let utah = document.querySelector("#nba134889");
-utah.addEventListener("click", locateTeamData);
-
-let goldenState = document.querySelector("#nba134865");
-goldenState.addEventListener("click", locateTeamData);
-
-let losAngelesC = document.querySelector("#nba134866");
-losAngelesC.addEventListener("click", locateTeamData);
-
-let losAngelesL = document.querySelector("#nba134867");
-losAngelesL.addEventListener("click", locateTeamData);
-
-let phoenix = document.querySelector("#nba134868");
-phoenix.addEventListener("click", locateTeamData);
+});
 
-let sacramento = document.querySelector("#nba134869");
-sacramento.addEventListener("click", locateTeamData);
+function addTeamConferences(item) {
+	if (
+		item.strTeamShort == "DEN" ||
```

Comment:
Since this is for categorizing these teams based on information the endpoint is not giving you, I understand why this is happening.

However, I have 2 suggestions for all of these chained OR conditions.

1. Create global variables out of these named `isWestern` for examples, that way your function will look something more like this
```js
function addTeamConferences(item){
  if(isWestern){
    item.strConference = "Western"
  } else {
    item.strConference = "Eastern"
  }
}
```
The benefit to this is that it will be immediately clear what the context of the condition is for.

2. Store these hardcoded strTeamShort values you're looking for into arrays and search for them in your condition
For example
```js
const westernTeams = ["DEN", "MIN", OKC, ...];
...
if (westernTeams.includes(item.strTeamShort)){
  item.strConference = "Western";
} else {
  item.strConference = "Eastern";
}
```

These are all recommendations based off of improving code maintainability and readability.

Reply:
Updated this and the function(s) to add the "Divisions" as well as the "Conferences"